modules/timepoint_list/locale/timepoint_list.pot

```diff
@@ -27,6 +36,9 @@ msgstr ""
 msgid "BVL QC"
 msgstr ""
 
+msgid "BVL QC Status"
+msgstr ""
```

Comment:
What's the difference between "BVL QC" and "BVL QC Status"? Maybe we should just stick to one?